docs/sources/installation/upgrading.md

```diff
@@ -123,7 +123,18 @@ If you're using systemd and have a large amount of annotations consider temporar
 If you have text panels with script tags they will no longer work due to a new setting that per default disallow unsanitized HTML.
 Read more [here](/installation/configuration/#disable-sanitize-html) about this new setting.
 
-### Authentication and security
+## Upgrading to v6.x
```

Comment:
@torkelo mentioned it would be nice to add something here. Not sure though if it makes sense to include it in this PR or add it later when creating a build

Reply:
Either add it here or create a separate issue for that doesn't matter.

Reply:
Think here makes sense

Reply:
In that case is there some release TODO where I can add "Change v6.x to current release in upgrading.md" or something like that?

Reply:
Not sure I follow. You want to change the header Upgrading to v6.x? This is targetet for 6.2 milestone so use Upgrading to v6.2.

Reply:
Hmm yes that makes sense though if it does not make it into 6.2 it will have to be changed to 6.3 :). So just wondered if there is some place to track it or maybe some label for the PR to use, marking that there is some release version dependant stuff that needs to be updated in case the release changes.

Reply:
You have a good point. Normally we happen to spot that upgrade instructions are needed in the end of a release cycle :smile:  

In this case I would create an issue for "Update upgrade notes" and put in 6.2 milestone and this is something we'll do as a task before release similar to writing What's new in and blog post. So to not lose this text I would suggest opening a separate PR targeting new "Update upgrade notes" issue.

Sounds good?

Reply:
Not sure if exactly what you mean but I created issue https://github.com/grafana/grafana/issues/16524 in the 6.2 milestone to remind us about this and changed the version here to 6.2 as that I would say is most likely. I think this should be enough to not forget about this.

Reply:
Sound good :+1: